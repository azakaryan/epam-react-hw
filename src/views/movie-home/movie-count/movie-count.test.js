import React from 'react';
import { shallow } from 'enzyme';
import expectEnzymify from 'expect';
import enzymify from 'expect-enzyme';
import MovieCount from './movie-count';

expectEnzymify.extend(enzymify());

describe('Movie Count Component', () => {
  it('should render correctly', () => {
    const component = shallow(<MovieCount count="2" />);

    expect(component).toMatchSnapshot();
  });

  it('renders a movie count component', () => {
    const wrapper = shallow(<MovieCount count="5" />);
    expectEnzymify(wrapper).toBeA('div');
  });

  it('renders a movie count component with correct count', () => {
    const wrapper = shallow(<MovieCount count="5" />);

    expectEnzymify(wrapper.find({ id: 'count' }).text()).toEqual('5');
  });
});
