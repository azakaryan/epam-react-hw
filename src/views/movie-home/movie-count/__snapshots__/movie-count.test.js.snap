// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Count Component should render correctly 1`] = `
<div
  className="main_container"
>
  <label
    className="count"
    id="count"
  >
    2
  </label>
  <span>
    movies found
  </span>
</div>
`;
